src/components/mgt-person/mgt-person.ts

```diff
@@ -445,6 +444,35 @@ export class MgtPerson extends MgtTemplatedComponent {
       } else {
         const initials = this.getInitials();
 
+        // work around to import fabric ui core colors without creating .d.ts file
+        const colors = [
```

Comment:
This list should be factored out to a static method so you don't recreate it each time on render